src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -195,6 +211,16 @@ protected void writeToNBT(NBTTagCompound compound)
         }
         compound.setTag(TAG_BUILDINGS, buildingTagList);
 
+        // Fields
+        NBTTagList fieldTagList = new NBTTagList();
+        for (Field f : fields.values())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'f' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)